db_release_scripts/mosip_pms/sql/1.2.1_pms-scripts_revoke.sql

```diff
@@ -16,4 +16,8 @@ ALTER TABLE pms.partner_h DROP COLUMN logo_url;
 
 ALTER TABLE pms.partner DROP COLUMN addl_info;
 
-ALTER TABLE pms.partner_h DROP COLUMN addl_info;
\ No newline at end of file
+ALTER TABLE pms.partner_h DROP COLUMN addl_info;
+
+UPDATE pms.partner_type SET is_active=false,upd_dtimes =now(), upd_by ='superadmin' WHERE code ='MISP_Partner'
```

Comment:
is it ok to activate all? What if someone has deactivated a partner earlier?

Reply:
it is not is_active, it should be is_policy_required set to false in this revoke script.

Reply:
We will be raising a new PR with the change sasi.